client/plots/survival.js

```diff
@@ -148,6 +148,13 @@ class TdbSurvival {
 								{ label: 'Tick', value: 'vtick' }
 							]
 						},
+						{
+							label: 'Survival Time Cut-Off',
+							type: 'number',
+							chartType: 'survival',
+							settingsKey: 'maxSurTime',
+							title: 'The max survival time to be displayed in this plot'
```

Comment:
@congyu-lu The title should also say that if left empty or `0`, would default to not having a max time-to-event cutoff.

Reply:
thanks just changed